### PR TITLE
Add tests for generic page processor

### PR DESCRIPTION
**Pull request by nileema**

### CODE REVIEW

File: presto-main/src/test/java/com/facebook/presto/operator/TestGenericPageProcessor.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        Page outputPage = processor.processColumnar(SESSION, page, types);
+        assertEquals(outputPage.getPositionCount(), page.getPositionCount());
+    }
+
+    @Test
+    public void testProcessColumnarDictionary()
+            throws Exception
+    {
+        Page page = createPage(types, true);
+        Page outputPage = processor.processColumnar(SESSION, page, types);
```
Comment (nezihyigitbasi) [inline presto-main/src/test/java/com/facebook/presto/operator/TestGenericPageProcessor.java]: Should this be `processor .processColumnarDictionary`?

File: presto-main/src/test/java/com/facebook/presto/operator/TestGenericPageProcessor.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    private final Page page = createPage(types, false);
+    private final PageBuilder pageBuilder = new PageBuilder(types);
+
+    @Test
+    public void testProcess()
+            throws Exception
+    {
+        processor.process(SESSION, page, 0, page.getPositionCount(), pageBuilder);
+        Page outputPage = pageBuilder.build();
+        assertEquals(outputPage.getPositionCount(), page.getPositionCount());
```
Comment (dain) [inline presto-main/src/test/java/com/facebook/presto/operator/TestGenericPageProcessor.java]: Can we use one of the test assertion methods to verify the contents of the output page?

**Comment by nileema** [comment]
@dain Updated

**Comment by dain** [comment]
loosk good

**Pull request merged**